src/backend/web/routes/feeds.js

```diff
@@ -72,4 +74,25 @@ feeds.post('/', protect(), async (req, res) => {
   }
 });
 
+feeds.delete('/:id', protect(), async (req, res) => {
+  const { user } = req;
+  const { id } = req.params;
+  try {
+    const feed = await Feed.byId(id);
+    if (!feed) {
+      return res.status(404).json({ message: `Feed for Feed ID ${id} doesn't exist.` });
+    }
+    if (user.owns(feed)) {
```

Comment:
Minor nit: I would deal with all the "fail" cases first, then do the "success" case last.  That would mean reversing your logic here:

```js
if(!user.owns(feed) {
  // 403
}
// delete and 204
```